recipes/deploy-worker.rb

```diff
@@ -86,7 +86,7 @@
     )
     install_live_streaming_service_config(most_recent_deploy, live_stream_name)
     install_published_event_details_email(most_recent_deploy, public_engage_hostname)
-    configure_newrelic(most_recent_deploy, newrelic_app_name)
+    configure_newrelic(most_recent_deploy, newrelic_app_name, :worker)
```

Comment:
Just to be consistent with the rest of the layer configuration this should be `:workers`